koin-projects/koin-android/src/main/java/org/koin/android/ext/android/ApplicationExt.kt

```diff
@@ -1,89 +0,0 @@
-/*
```

Comment:
Why did you removed this ? What about backward compat ? Is it mandatory ?

Reply:
I've upgrade the extension to ComponentCallbacks.statrKoin(), which is equivalent to old Application.statrKoin(). But it allows to use this startKoin from another Android class if needed (ContentProvider for example)